openlibrary/plugins/upstream/borrow.py

```diff
@@ -118,13 +118,11 @@ def POST(self, key):  # noqa: PLR0915
         i = web.input(
             action='borrow',
             format=None,
-            ol_host=None,
```

Comment:
I thought `ol_host` is still being used to indicate a loan intent was initiated from openlibrary.